appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/eia-environmental-impact-schedule/eia-environmental-impact-schedule.controller.js

```diff
@@ -0,0 +1,85 @@
+import logger from '#lib/logger.js';
+import {
+	changeEiaEnvironmentalImpactSchedulePage,
+	mapEiaEnvironmentalImpactScheduleRadioValueForService
+} from './eia-environmental-impact-schedule.mapper.js';
+import { changeEiaEnvironmentalImpactSchedule } from './eia-environmental-impact-schedule.service.js';
+import { getLpaQuestionnaireFromId } from '../lpa-questionnaire.service.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const getChangeEiaEnvironmentalImpactSchedule = async (request, response) => {
```

Comment:
Is this just an alias?

Reply:
Yep - refactored to reference `renderChangeEiaEnvironmentalImpactSchedule` directly now